src/ingest/src/main/scala/osmesa/ingest/Ingest.scala

```diff
@@ -122,21 +126,45 @@ object IngestApp extends CommandApp(
 
       val df = ss.read.orc(orc)
 
-      val ppnodes = ProcessOSM.preprocessNodes(df)
-      val ppways = ProcessOSM.preprocessWays(df)
-      val nodeGeoms = ProcessOSM.constructPointGeometries(ppnodes)
-      val wayGeoms = ProcessOSM.reconstructWayGeometries(ppnodes, ppways)
-      println("PRIOR TO WAY/NODE UNION")
-      nodeGeoms.withColumn("minorVersion", lit(null).cast(IntegerType)).printSchema
-      wayGeoms.printSchema
+      println("caching...")
+      val cache = Option(new URI(cacheDir).getScheme) match {
+        case Some("file") => Caching.onFs(cacheDir)
+        case Some("s3") => Caching.onS3(cacheDir)
+        case _ => Caching.none
+      }
+
+      val orcFileRepr = orc.split("/").last.split('.').head
+
+      val clippedWorldExtent = Extent(-180, -75, 180, 85).reproject(LatLng, WebMercator)
```

Comment:
-75?

Reply:
This may be nonstandard, but there are geometries in Antarctica that mess up our WebMercator projections that need to be pulled.

Reply:
Maybe better to start in web mercator to get all the area we can?:

```
val EARTH_RADIUS = 6378137
val clippedWorldExtent = Extent(-Math.PI * EARTH_RADIUS, -Math.PI * EARTH_RADIUS, Math.PI * EARTH_RADIUS, Math.PI * EARTH_RADIUS)
```

Reply:
Ah, interesting. Can you add a note about that (ideally w/ a link)?

Reply:
(in the source, that is)